apis/Google.Cloud.PubSub.V1/docs/index.md

```diff
@@ -82,6 +82,99 @@ restart listening for messages with `StartAsync(...)` again. Due to the expense
 instance, it is recommended that a singleton client per topic is used for the lifetime of the
 application.
 
+## Dependency Injection
+
+Both `PublisherClient` and `SubscriberClient` can be easily integrated with the [dependency injection](https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection)
+container provided by the [Microsoft.Extensions.DependencyInjection](https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection/) package.
+The `Google.Cloud.PubSub.V1` package provides extension methods to register the clients with the dependency
+injection container in the `Microsoft.Extensions.DependencyInjection` namespace.
+
+### PublisherClient
+
+To register a singleton `PublisherClient` instance with default settings in the `IServiceCollection`, use the
+`AddPublisherClient` extension method as shown below:
+
+{{sample:PublisherClient.AddPublisherClient}}
+
+There is an overload of `AddPublisherClient` method that takes `Action<PublisherClientBuilder>` as a parameter
```

Comment:
Same comment as I've written below for AddSubscriberClient. (I just missed it when reading this the first time.)